components/cache/adapters/chain_adapter.rst

```diff
@@ -17,14 +17,13 @@ lifetime as its constructor arguments::
 
     use Symfony\Component\Cache\Adapter\ApcuAdapter;
```

Comment:
I know it's not related to your change, but I think there is wrong import. What do you think about removing 
```
use Symfony\Component\Cache\Adapter\ApcuAdapter;
```
and including:
```
use Symfony\Component\Cache\Adapter\ChainAdapter;
```
?

Reply:
You are right @gregurco, I changed it while merging.